### DISCUSSION TITLE
Where is the release?

### DISCUSSION THREAD

**Discussion by davlod** [stustapay/stustapay#399] [Q&A]
Hi, I have a question. Because trying to install everything the only thing I miss is the .deb ... But where exactly would the release be in your directory? Because looking refinedly we could not find anything, in this regard I as the heck install.

**Comment by TheJJ** [comment]
the deb files are artifacts of the CI run, but we don't have a stable release deb yet.
you can build the `.deb` yourself with the steps from the github action:

https://github.com/stustapay/stustapay/blob/5c9c579fd011d8430752fcb9c197280d24f1332b/.github/workflows/push_on_master.yaml#L49

the ci pipeline defines how the deb is build - which you can do easily on your device.

what for do you need the release directly? what's your application of stustapay?

**Comment by tonowie** [comment]
if you are looking for quick way to start things up for testing you can use [docker compose from my fork](https://github.com/tonowie/stustapay/tree/compose/compose). Readme should get you through the setup like certificates for https and domain. All runs fine on my laptop without need to install any dependencies outside of docker
